boot/core/src/boot/task_helpers.clj

```diff
@@ -35,21 +35,11 @@
         (->> sym ns-aliases (into base) (mapcat pubs)))
       (filter task?) (sort-by :name) (group-by :ns*) (into (sorted-map)))))
 
-(defn read-pass
-  [prompt]
-  (String/valueOf (.readPassword (System/console) prompt nil)))
-
```

Comment:
We may want to either keep this or move it to `boot.util`? It seems like a handy function to have hanging around.